FishjamClient/src/main/java/org/membraneframework/rtc/error/MembraneError.kt

```diff
@@ -0,0 +1,5 @@
+package org.membraneframework.rtc.error
+
+enum class MembraneError(val message: String) {
+    TRACK_NOT_READY("Track was not confirmed by backed. Try again later")
```

Comment:
I'd change this message to something like "Track updated while being negotiated in peer connection. Wait for the track to be negotiated.". "Track was not confirmed by the backend" and "Try again later" is too vague imo. 
Now that I think about it, how user should wait for the negotiation?